handlers/EveryoneAndHereMentionMessage.js

```diff
@@ -0,0 +1,30 @@
+const Discord = require('discord.js');
+const MentionsList = [
```

Comment:
Tu bardziej niż tablica pasowało by użyć obiektu, ponieważ jest to jakaś mapa słownikowa. Łatwiej wtedy odczytywać wartości z takiego słownika, bo zamiast `MentionsList[0]` robi się `MentionsList.EVERYONE`, co nawet IDE może podpowiedzieć. Tablice nadają się do struktur o uporządkowanej kolejności, a tutaj niwelujemy _magic stringi_ do postaci reużywalnych zmiennych.

W TypeScript tu nadał by się enum (który i tak [wyjściowo jest obiektem](https://www.typescriptlang.org/docs/handbook/enums.html#enums-at-runtime)).